docs/src/udp.rst

```diff
@@ -79,6 +79,11 @@ Data types
              * Indicates that recvmmsg should be used, if available.
              */
             UV_UDP_RECVMMSG = 256
+
+            /*
+             * Indicates that receive an ICMP error message, if available.
+             */
+            UV_UDP_RECVERR = 512
```

Comment:
Maybe just reuse the UV_UDP_LINUX_RECVERR flag instead of defining a new one?

Reply:
Done !